pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/crossClusterReplication.tsx

```diff
@@ -48,5 +49,32 @@ export default function (props: GraphDashboardProps) {
         />
       </Axis>
     </LineGraph>,
+    <LineGraph
+      title="Replication Lag"
+      sources={storeSources}
+      tenantSource={tenantSource}
+      tooltip={`Replication lag between primary and standby cluster`}
+    >
+      <Axis units={AxisUnits.Duration} label="duration">
+        <Metric
+          downsampler={TimeSeriesQueryAggregator.MIN}
+          aggregator={TimeSeriesQueryAggregator.AVG}
+          name="cr.node.physical_replication.replicated_time_seconds"
+          title="Replication Lag"
+          transform={datapoints =>
+            datapoints.map(d =>
```

Comment:
`Do we need to wrap this in a `map`? `transform` seems to be equivalent to `map` so we can just pass the inner function here.

Reply:
There's a use-case where we may want to do more than just a `map`. For instance, we just finished up some discussion on the replication lag metric where we want to filter out `replicated_time_seconds` with a value of `0` (as during the initial scan phase).

We could add separate props for `filter` and `map`, but I figured that having a general purpose `transform` may be better. Thoughts?

Reply:
Gotcha. I don't have a strong preference here so sgtm!